HW05.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: "HW05"
+author: "Ricardo Muñiz Trejo"
+date: "2/8/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(knitr)
+library(tidyverse)
+```
+
+## Tidying the dad_mom file
+
+Untidied version of the dad_mom data frame:
+
+
+```{r}
+dad_mom_messy <- read_tsv("dad_mom.txt")
+
+kable(dad_mom_messy, format = "markdown")
+```
+
+Tidied version of the dad_mom data frame:
+
+```{r}
+
+#Ok, this one can be confusing. The objective is to have 4 variables: fam_id, name, parent, income
+
+dad_mom_tidy <- dad_mom_messy %>%
+  #First we need to create the parent variable which considers if its the mom or the dad
+  separate(col = name_dad, into = c("name_dad", "parent_dad")) %>%
+  separate(col = name_mom, into = c("name_mom", "parent_mom")) %>%
+  replace_na(list(parent_dad = "dad",
```

Comment:
This is a very creative solution to create the parent column. I considered something like this in mine but got stuck on how to track mom to the moms and dad to the dads. Making them their own columns and then combining them is smart